node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs');
+
+fs.readFile('hopper.txt', 'utf8', (err, data) => {
```

Comment:
This one should be able to read any file so you are going to need to use `process.argv`.

Reply:
![Screen Shot 2021-03-10 at 5 20 09 PM](https://user-images.githubusercontent.com/74885918/110721361-40c31800-81c5-11eb-83f3-1ea63f3b0967.png)